### PR TITLE
Update tracing-subscriber to 0.3

### LINKED ISSUE #469: Update examples to tracing-subscriber 0.3
I believe this will require enabling the `env-filter` feature.

### PR DESCRIPTION
**Pull request by EdorianDark**
Fixes #469

### CODE REVIEW

**Comment by EdorianDark** [comment]
I probably forgot something, this was too easy.

**Comment by davidpdrsn** [comment]
Did you try running one of the examples to verify that it still works? I used tracing-subscriber 0.3 a bit the other day and had to enable the `env-filter` feature to get it working with `RUST_LOG`. I might have missed something obvious however.

**Comment by EdorianDark** [comment]
``RUST_LOG``works now

File: examples/async-graphql/Cargo.toml [hunk 1]
```diff
@@ -8,6 +8,6 @@ publish = false
 axum = { path = "../../axum" }
 tokio = { version = "1.0", features = ["full"] }
 tracing = "0.1"
-tracing-subscriber = "0.2"
+tracing-subscriber = {version="0.3", features=["env-filter"]}
```
Comment (davidpdrsn) [inline examples/async-graphql/Cargo.toml:11]: nitpick: We normally format the dependencies like this

```suggestion
tracing-subscriber = { version="0.3", features=["env-filter"] }
```

Comment (EdorianDark) [reply examples/async-graphql/Cargo.toml]: Ok, done

**Review by davidpdrsn — APPROVED** [review]
Perfect 👌 thank you!

**Pull request merged**